### DISCUSSION TITLE
Plugged in all the time/overnight?

### DISCUSSION THREAD

**Discussion by splnkr** [AppHouseKitchen/AlDente-Battery_Care_and_Monitoring#80] [Q&A] [answered]
Hey,
first of all, thanks for the program. It works good on my mid 2014 macbook pro.
So to my question: is it okay, if i leave my mac plugged in all the time? 

Kind regards,
splnkr

**Comment by MatthiasKerbl** [comment, marked as answer]
Hey splnkr,
Your battery should be fine being plugged in all the time. Maybe unplug the Macbook once a month and do 1-2 full charging cycles to rebalance the battery.

**Reply by splnkr** [reply]
Ah that's nice. Do you mean by doing a full cycle charging from 20%/5%/0%(dead battery) to 100%?

**Reply by MatthiasKerbl** [reply]
You should discharge completely to 0%, then charge to 100% without interruptions and leave it plugged in for at least 30 minutes when it's at 100%. Afterward, you can set AlDente to the desired SoC and drain the battery until you reach this SoC.
